index.html

```diff
@@ -51,6 +53,238 @@ <h3>LET’S CONNECT</h3>
+        </div>
+        <!-- End of fourth card -->
+      </section>
+      <!-- End of Work Section -->
+
+      <!-- ABOUT MYSELF SECTION -->
+      <section id="aboutmyself">
+        <div class="top">
+          <div class="h2">About <br />Myself</div>
+          <div class="para">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </div>
+          <div class="social-media">
+            <div class="supporting-text">
+              <div class="h3">LET’S CONNECT</div>
+            </div>
+            <div class="social-media-icons">
+              <img
+                src="./images/social/github.svg"
+                alt="github icon"
+                class="social-icon"
+              />
+              <img
+                src="./images/social/linkedin.svg"
+                alt="linkedin icon"
+                class="social-icon"
+              />
+              <img
+                src="./images/social/twofinger.svg"
+                alt="two finger icon"
+                class="social-icon"
+              />
+              <img
+                src="./images/social/twitter.svg"
+                alt="twitter icon"
+                class="social-icon"
+              />
+              <img
+                src="./images/social/mail.svg"
+                alt="mail icon"
+                class="social-icon"
+              />
+            </div>
+          </div>
+          <div class="action-box">
+            <div class="button">Get my resume</div>
+          </div>
+        </div>
+        <!-- End of top -->
+        <div class="skillset">
+          <div class="languages">
+            <h2>Languages</h2>
+            <img
+              src="./images/chevron/Icon_Chevron_Down_Larger.png"
+              alt="Chevron Down"
+            />
+          </div>
+          <div class="language-list">
+            <div class="lang">
+              <img
+                src="./images/languages/javascript.png"
+                alt="Javascript Image"
+              />
+              <h3>Javascript</h3>
+            </div>
+            <div class="lang">
+              <img src="./images/languages/html.png" alt="HTML Image" />
+              <h3>HTML</h3>
+            </div>
+            <div class="lang">
+              <img src="./images/languages/css.png" alt="CSS Image" />
+              <h3>CSS</h3>
+            </div>
+          </div>
+          <div class="frameworks">
+            <h2>Frameworks</h2>
+            <img src="./images/chevron/Disabled.png" alt="Chevron Disabled" />
+          </div>
+          <div class="skills">
+            <h2>Skills</h2>
+            <img src="./images/chevron/Disabled.png" alt="Chevron Disabled" />
+          </div>
+        </div>
+        <!-- End of SkillSet -->
```

Comment:
According to the project requirements, this content should be wrapped inside a `<ul>` tag and each category should be wrapped inside a `<li>` tag. Additionally, the list of languages needs to be wrapped inside a `<ul>` tag with each language being wrapped in a `<li>` tag as well.

<img width="747" alt="Screen Shot 2022-05-27 at 17 48 20" src="https://user-images.githubusercontent.com/82426682/170797828-7c653656-ff7a-41a4-a792-bc5971d2d315.png">